pkg/operator/controller/controller_lb.go

```diff
@@ -19,9 +19,46 @@ import (
 const (
 	// awsLBProxyProtocolAnnotation is used to enable the PROXY protocol on any
 	// AWS load balancer services created.
+	//
+	// https://kubernetes.io/docs/concepts/services-networking/service/#proxy-protocol-support-on-aws
 	awsLBProxyProtocolAnnotation = "service.beta.kubernetes.io/aws-load-balancer-proxy-protocol"
 )
 
+var (
+	// internalLBAnnotation maps platform to the annotation name and value
+	// that tell the cloud provider that is associated with the platform
+	// that the load balancer is internal.
+	//
+	// https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+	internalLBAnnotation = map[configv1.PlatformType][]string{
```

Comment:
I found this structure a little weird compared to:

```go
internalLBAnnotation = map[configv1.PlatformType]map[string]string{
  configv1.AWSPlatformType: {
    "service.beta.kubernetes.io/aws-load-balancer-proxy-protocol": "*",
  },
  // ...
}
```

Which then gives the caller a map to merge with any other label/annotation maps.

But, what you have works, so I just mention it as a matter of style to consider.

Reply:
Yeah, I know the string slice is weird.  Using a map seems silly in abstract, but it Go's syntactic sugar makes it look nicer.  Changed.